test/features/schema_tests.js

```diff
@@ -11,6 +11,11 @@ var Ajv = require('ajv');
 describe('Responses should conform to the provided JSON schema of the responce', function() {
     var ajv = new Ajv({});
 
+    function getToday() {
+        const now = new Date();
+        return `${now.getUTCFullYear()}/${now.getUTCMonth() + 1}/${now.getUTCDate()}`;
```

Comment:
I'd propose to use ISO timestamps instead. Those sort lexicographically, and are easier to parse.